assets/src/edit-story/output/test/page.js

```diff
@@ -405,6 +410,132 @@ describe('Page output', () => {
     });
   });
 
+  describe('pageAttachment', () => {
```

Comment:
Some AMP validation tests would be nice

Reply:
Added a validation test for when page attachment is present.